test/helpers/sharedSteps.ts

```diff
@@ -15,94 +15,100 @@
 import {
   DEFAULT_DEAL_NAME,
   FLUENCE_CONFIG_FULL_FILE_NAME,
   FS_OPTIONS,
-  LOCAL_NET_DEFAULT_WALLET_KEY,
   RUN_DEPLOYED_SERVICES_FUNCTION_CALL,
+  RUN_DEPLOYED_SERVICES_FUNCTION_NAME,
+  type Template,
   WORKER_SPELL,
-} from "../src/lib/const.js";
+} from "../../src/lib/const.js";
 import {
-  jsonStringify,
   LOGS_GET_ERROR_START,
   LOGS_RESOLVE_SUBNET_ERROR_START,
   setTryTimeout,
   stringifyUnknown,
-} from "../src/lib/helpers/utils.js";
-import { getServicesDir, getSpellsDir, getSrcPath } from "../src/lib/paths.js";
+} from "../../src/lib/helpers/utils.js";
+import { addrsToNodes } from "../../src/lib/multiaddres.js";
+import { getAquaMainPath } from "../../src/lib/paths.js";
+import { validateDeployedServicesAnswerSchema } from "../validators/deployedServicesAnswerValidator.js";
+import { validateSpellLogs } from "../validators/spellLogsValidator.js";
+import {
+  validateWorkerServices,
+  type WorkerServices,
+} from "../validators/workerServiceValidator.js";
 
+import { fluence } from "./commonWithSetupTests.js";
 import {
-  multiaddrs,
+  fluenceEnv,
   MY_SERVICE_NAME,
   RUN_DEPLOYED_SERVICES_TIMEOUT,
 } from "./constants.js";
-import { assertHasWorkerAndAnswer, fluence } from "./helpers.js";
-import { validateDeployedServicesAnswerSchema } from "./validators/deployedServicesAnswerValidator.js";
-import { validateSpellLogs } from "./validators/spellLogsValidator.js";
+import { TEST_DEFAULT_SENDER_ACCOUNT } from "./localNetAccounts.js";
 import {
-  validateWorkerServices,
-  type WorkerServices,
-} from "./validators/workerServiceValidator.js";
-
-export function getServiceDirPath(cwd: string, serviceName: string) {
-  return join(getServicesDir(cwd), serviceName);
+  getInitializedTemplatePath,
+  getMainRsPath,
+  getModuleDirPath,
+  getServiceDirPath,
+  getSpellAquaPath,
+  getSpellDirPath,
+  pathToTheTemplateWhereLocalEnvironmentIsSpunUp,
+} from "./paths.js";
+
+export const multiaddrs = await getMultiaddrs();
+
+export async function getMultiaddrs(): Promise<Node[]> {
+  const local =
+    fluenceEnv === "local"
+      ? addrsToNodes(
+          (
+            await fluence({
+              args: ["default", "peers", "local"],
+              cwd: pathToTheTemplateWhereLocalEnvironmentIsSpunUp,
+            })
+          )
+            .trim()
+            .split("\n"),
+        )
+      : [];
+
+  return {
+    kras: krasnodar,
+    stage: stage,
+    testnet: testNet,
+    local,
+  }[fluenceEnv];
 }
 
-export function getModuleDirPath(
+export const initializeTemplate = async (
   cwd: string,
-  moduleName: string,
-  serviceName?: string,
-) {
-  if (serviceName === undefined) {
-    return join(getSrcPath(cwd), "modules", moduleName);
-  }
+  template: Template,
+): Promise<void> => {
+  const templatePath = getInitializedTemplatePath(template);
 
-  return join(getServiceDirPath(cwd, serviceName), "modules", moduleName);
-}
+  try {
+    await rm(cwd, { recursive: true });
+  } catch {}
```

Comment:
```suggestion
  await rm(cwd, { recursive: true, force: true });
```

Acording to Node.JS [docs](https://nodejs.org/api/fs.html), the force option here will prevent throwing the error.